lib/resolveToggles.js

```diff
@@ -1,4 +1,4 @@
-module.exports = async function (toggles) {
+module.exports = function (toggles) {
```

Comment:
I removed this `async` since there's no `await` in the block, it was unnecessary